### DISCUSSION TITLE
Global Low Battery Threshold

### DISCUSSION THREAD

**Discussion by SergeantPup** [andrew-codechimp/HA-Battery-Notes#1555] [Ideas]
I've discovered the low battery threshold on each device. Is there a way to configure this globally rather than every individual device? I have 107 devices with batteries and I'm just going to configure them all to 25%. I didn't see this in the documentation or other discussions, my apologies if this is already posted somewhere. If not, I'm recommending an idea for this. 

Globally allow me to overwrite the low battery threshold on all devices, then if there's certain devices I want different, I can go in and update those manually.

**Comment by andrew-codechimp** [comment]
There is a [configuration setting](https://andrew-codechimp.github.io/HA-Battery-Notes/configuration/#configuration) that allows this.

```
battery_notes:
  default_battery_low_threshold: 10
```
